config/gnu-warnings/noerror-general

```diff
@@ -0,0 +1,91 @@
+#
+# HDF5 code should not trigger the following warnings under any
+# circumstances, so ask the compiler to treat them as errors:
```

Comment:
This comment seems wrong in noerror* files, if I understand the difference between error and noerror files.  The lines in the list that follows begin with -Werror= in the error-general file, so I assume that with the no-error file we aren't asking the compiler to treat them as errors.  Probably the comment was copied and pasted from the error file, but for someone coming along later trying to figure out what these files do, if the comment is incorrect and misleading, it will make understanding the purpose more difficult.

Reply:
Probably, but it needs to be fixed in develop first.
These merges are to move the changes already in develop to a branch.
An issue should be created.